.github/ISSUE_TEMPLATE/01-bug.yml

```diff
@@ -249,6 +249,12 @@ body:
         - *SF_TF_GOSNOWFLAKE_LOG_LEVEL=debug* - if you feel the problem may be connected with the Snowflake driver, please set this one for additional driver logs.
       placeholder: https://gist.github.com/example/12345678
 
+  - id: company-name
```

Comment:
Two remarks:
1. let's put it much closer to the top (maybe even in the `Terraform CLI and Snowflake Provider Versions` section)
2. Let's rephrase to: `Optional field. Providing this information allows us to communicate faster which may lead to faster issue resolution.`

Reply:
Done both.